packages/algob/sample-project/ts/algob.config.ts

```diff
@@ -0,0 +1,89 @@
+// NOTE: below we provide some example accounts.
+// DON'T this account in any working environment because everyone can check it and use
+// the private keys (this accounts are visible to everyone).
+
+// NOTE: to be able to execute transactions, you need to use an active account with
+// a sufficient ALGO balance.
+
+/**
+	 Check our /docs/algob-config.md documentation (https://algobuilder.dev/guide/algob-config.html) for more configuration options and ways how to
+	load a private keys:
+	+ using mnemonic
+	+ using binary secret key
+	+ using KMD daemon
+	+ loading from a file
+	+ loading from an environment variable
+	+ ...
+*/
+
+// ## ACCOUNTS USING mnemonic ##
+const { mkAccounts, algodCredentialsFromEnv } = require("@algo-builder/algob");
+let accounts = mkAccounts([
+	{
+		// This account is created using `make setup-master-account` command from our
+		// `/infrastructure` directory. It already has many ALGOs
+		name: "master",
+		addr: "WWYNX3TKQYVEREVSW6QQP3SXSFOCE3SKUSEIVJ7YAGUPEACNI5UGI4DZCE",
+		mnemonic:
+			"enforce drive foster uniform cradle tired win arrow wasp melt cattle chronic sport dinosaur announce shell correct shed amused dismiss mother jazz task above hospital",
+	},
+]);
+
+// ## ACCOUNTS loaded from a FILE ##
+// const { loadAccountsFromFileSync } = require("@algo-builder/algob");
+// const accFromFile = loadAccountsFromFileSync("assets/accounts_generated.yaml");
+// accounts = accounts.concat(accFromFile);
+
+/// ## Load accounts from KMD ##
+/// Please check https://github.com/scale-it/algo-builder/blob/master/docs/guide/algob-config.md#network-credentials for more details and more methods.
+//   let kmdCred = KMDCredentialsFromEnv();
+
+let defaultCfg = {
```

Comment:
Here I think you can make `defaultCfg` as type `Network` or something like that

Reply:
We can't import this type outside the `sample-project` folder.